### PR TITLE
Updates after review №1

### PR DESCRIPTION
**Pull request by tiutiunnykleasoft**
Last PR was closed because of force-push, sorry for that :(

### CODE REVIEW

File: src/Entities/Client.php [hunk 1]
```diff
@@ -13,47 +13,47 @@ final class Client implements MultiFieldsEntityInterface
...
+        string $platformName,
+        string $platformVersion,
+        string $pluginName,
+        string $pluginVersion
     )
     {
         $this->user_agent = $this->createSimpleField(
-            property_name: 'user_agent',
+            propertyName: 'user_agent',
             value: $user_agent
```
Comment (BohdanLeasoft) [inline src/Entities/Client.php:40]: Should is be in PSR like `$userAgent`?

File: src/Entities/Client.php [hunk 2]
```diff
@@ -64,21 +64,21 @@ public function __construct(
```
Comment (BohdanLeasoft) [inline src/Entities/Client.php:64]: Should it be `return $this->userAgent->get()`?

File: src/Tests/CreateOrderTest.php [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    public function setUp(): void
+    {
+        $_SERVER["REMOTE_ADDR"] = "173.0.2.5";
+        $_SERVER["HTTP_USER_AGENT"] = "PHPUnit Tests";
+
+        $this->client = new \GingerPluginSdk\Client(
+            new \GingerPluginSdk\Properties\ClientOptions(
+                endpoint: "https://api.online.emspay.eu",
+                useBundle: true,
+                apiKey: getenv('GINGER_API_KEY'))
```
Comment (BohdanLeasoft) [inline src/Tests/CreateOrderTest.php:35]: Just question. For now it is not necessary, but is it possible in future make variable endpoint like apiKey? Then this tests could be used with different banks and their endpoints

File: src/Client.php [hunk 4]
```diff
@@ -42,8 +40,60 @@ public function createClient($apiKey, $useBundle, $endpoint): ApiClient
...
             $useBundle ?
                 [
-                    CURLOPT_CAINFO => self::getCaCertPath()
+                    CURLOPT_CAINFO => self::CA_CERT_FILE_PATH
                 ] : []
         );
     }
+
+    public function checkAvailabilityForPaymentMethodUsingCurrency(string $payment_method_name, Currency $currency): bool
+    {
```
Comment (BohdanLeasoft) [inline src/Client.php:49]: Just a question. Will be added the ability to clear cached multicurrency?

Comment (tiutiunnykleasoft) [reply src/Client.php:49]: Good question, i think i can use it to make a method, which removes the cached file. We will be able to use this method as trigger to start update for caching multi currency which planing to use after save API key actions.

File: src/Entities/Client.php (hunk 2 above)
Comment (tiutiunnykleasoft) [reply src/Entities/Client.php:64]: yes, but how it now?

**Pull request merged**